pages/bridge.tsx

```diff
@@ -245,13 +245,13 @@ const Bridge = () => {
               <div className="mt-5">
                 {bridgeTarget === BridgeTarget.L2
                   ? bridgeToL2Error &&
-                    BigInt(amount) !== BigInt(0) && (
+                    parseEther(amount) !== BigInt(0) && (
```

Comment:
good call, but we should use `parseUnits` instead, because L1 gov tokens aren't guaranteed to have 18 decimals.

Reply:
actually, parsing here isn't so important. this is just the error boundary, and the part of the conditional we're editing here functions mainly as a "if the user hasn't touched the input yet..." check. Therefore, i think it would suffice to check the input value (`amount`) against its initial state (`'0'`) aka `amount !== "0"` instead. This goes for the other change, too. Let me know what you think.

Reply:
@garyghayrat and i synced and we will use `rawAmount` which is defined earlier in the file; longer term we want to make the bridge page smarter via the prepare hook's `enabled` flag (and/or investigate `suspense` flag)

Reply:
Thanks again for explaining me through the thought process! 

Implemented changes to use `rawAmount` in [abbee38](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/90/commits/abbee389c617ff9cb42ff93ea301963e1c1c4bf1)